packages/core/app/mirage/routes/report.js

```diff
@@ -3,22 +3,68 @@ import moment from 'moment';
 import RESPONSE_CODES from '../enums/response-codes';
 const TIMESTAMP_FORMAT = 'YYYY-MM-DD HH:mm:ss';
 
+/**
+ * @method getFilterParams
+ * @param {String} queryFilter
+ * @returns {Array} Filter parameters
+ * @description Parse filter parameters in the form of "(title==*H*,request==*Revenue*)";author==*ramvish*
+ * to a list of all the OR parameters, ie., [H, Revenue]
+ */
+const getFilterParams = function(queryFilter) {
+  return queryFilter
+    .split(';')[0]
+    .replace(/\(/g, '')
+    .replace(/\)/g, '')
+    .replace(/\*/g, '')
+    .split(',')
+    .map(el => el.split('==')[1]);
```

Comment:
Rather than doing all the replaces, can we just use a RegEx?
`return queryFilter.split(';')[0].match(/\*(.*?)\*/g).map(val => val.replace('*', ''));`